src/interfaces/store-interfaces/action-interface.ts

```diff
@@ -6,5 +6,5 @@ export interface ActionIF {
 }
 
 export interface ActionBindingIF {
-  (x: any | null, y?: any | null): any
+  (x: any | null, y?: any | null, z?: any | null): any
```

Comment:
Really? lol

Reply:
Oops sorry, i was testing something with actions and the way it handles additional parameters. 
Which it doesn't! So i will remove this :)

Reply:
Fixed!